Source/WebCore/page/Quirks.cpp

```diff
@@ -1716,6 +1716,21 @@ bool Quirks::shouldDisableNavigatorStandaloneQuirk() const
     return false;
 }
 
+// booking.com https://webkit.org/b/269875
+// FIXME: booking.com https://webkit.org/b/269876 when outreach has been successful.
+bool Quirks::shouldSendLongerAcceptHeaderQuirk(const URL& url, LocalFrame* frame)
+{
+    auto host = url.host();
```

Comment:
Please move this *after* the early return. No point in doing this if we're about to early return (common case).